### ISSUE TITLE
Bulk ticker endpoint

### ISSUE THREAD

**Issue by CarbonAlabel** [ccpgames/esi-issues#362]
An endpoint similar to `/universe/names/` for retrieving corporation/alliance tickers in bulk might be useful.

**Comment by Risingson** [comment]
i would like to see the ticker added to `/alliances/names/` as referenced.

**Comment by Risingson** [comment]
if you do not want to add it to `/alliances/names/` maybe do a `/alliances/tickers/` endpoint returning only `alliance_id` and `alliance_ticker` ?

**Comment by aquarhead** [comment]
Can you provide a use case for this? Is there anywhere from API/client that you can copy a list of tickers instead of alliance ids and/or alliance names?

We're not against this but 1. this has to be delayed until parity is done 2. having a solid use case would help us set the priority.

**Comment by Risingson** [comment]
I do a client-side map application. The app fetches sov structures (esi) and compiles a list of the alliance_ids holding them ready to be fed to /alliances/names/. 
I switched nearly all api calls to esi but still use the xml-api to display the alliance names and the ticker (shortName) for identification on the maps.

`/eve/AllianceList.xml.aspx?version=1`
`<row name="Goonswarm Federation" shortName="CONDI" allianceID="1354830081" executorCorpID="1344654522" memberCount="16042" startDate="2010-06-01 05:36:00">`

I get `name` and `shortName` with a single call. In my use-case having to fetch every single esi /alliances/{alliance_id}/ just to get the shortName is unnecessary bytes on the wire, delays and hurts user experience at app startup and just doesn't feel right as the ticker is a name aswell hence belonging to the /alliances/names/ endpoint imo.

Screenshot: https://imgur.com/YsDEc00
Link: https://eveeye.com/?m=Delve&s=W-KQPI&t=sov

**Comment by CarbonAlabel** [comment]
@aquarhead Just the tickers? No. However, corporate/alliance tickers are crucial parts of their identity, just as important as the name itself. I can think of a couple of corps which are almost exclusively referred to by their tickers.

This is very much a convenience endpoint, so I don't have a problem with it being delayed, but I'd still say it's a pretty important one. For a specific use case: client side local scan tool, entirely relying on ESI - for busier systems, could replace dozens or hundreds of calls with a single one. Just about any application which displays corporation and alliance affiliations of pilots would find this a significant QoL improvement.

**Comment by Slazanger** [comment]
I too would like this to match the bulk alliance id to name

**Comment by aquarhead** [comment]
Yeah, sorry, was thinking bulk ticker -> alliance id for some reason, a bulk id -> ticker should be useful for many cases.
